apps/gitness/src/pages-v2/repo/repo-import-page.tsx

```diff
@@ -24,10 +38,10 @@ export const ImportRepo = () => {
       provider: {
         host: data.hostUrl ?? '',
         password: data.password,
-        type: 'github',
+        type: data.provider.split(' ')[0].toLowerCase(),
```

Comment:
instead of string manipulation, we should have enums or keys per provider.

Reply:
done